romanesco/__init__.py

```diff
@@ -284,11 +287,13 @@ def extractId(spec):
 
             if auto_convert:
                 outputs[name] = romanesco.convert(
-                    task_output["type"], script_output, d, **kwargs)
+                    task_output["type"], script_output, d,
+                    task_output=task_output, **kwargs)
```

Comment:
This one does not seem consistent with all the other calls to `fetch`, `push`, `isvalid`, `convert` in this file.